usr/share/rear/lib/_input-output-functions.sh

```diff
@@ -403,6 +403,14 @@ function UserInput () {
     local timeout=300
     # Avoid stderr if USER_INPUT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_TIMEOUT:
     test "$USER_INPUT_TIMEOUT" -ge 0 2>/dev/null && timeout=$USER_INPUT_TIMEOUT
+    # Have some seconds (at least one second) delay when an automated user input is used to be fail-safe against
+    # a possibly false specified predefined user input value for an endless retrying loop of UserInput calls
+    # that would (without the delay) run in a tight loop that wastes resources (CPU, diskspace, and memory)
+    # and fills up the ReaR log file (and the disk - which is a ramdisk for 'rear recover')
+    # with some KiB data each second that may let 'rear recover' fail with 'out of diskspace/memory':
+    local automated_input_interrupt_timeout=3
+    # Avoid stderr if USER_INPUT_INTERRUPT_TIMEOUT is not set or empty and ignore wrong USER_INPUT_INTERRUPT_TIMEOUT:
+    test "$USER_INPUT_INTERRUPT_TIMEOUT" -ge 1 2>/dev/null && automated_input_interrupt_timeout=$USER_INPUT_INTERRUPT_TIMEOUT
```

Comment:
This variable should be set to a sane default in `default.conf` and then you can use `((` to check the value and abort on unexpected values.

If this  is something that users should set when running ReaR then I prefer a command line argument instead. Makes for much better documentation than magic secret environment variables.